Extension bundling breaks Docker build

The extension bundling from #1922 broke Docker build:

```
#0 6.992   cargo:warning=ToolExecError: Command "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "bundled/sqlean/" "-std=c11" "-DSQLITE_CORE" "-DSQLITE_DEFAULT_FOREIGN_KEYS=1" "-DSQLITE_ENABLE_API_ARMOR" "-DSQLITE_ENABLE_COLUMN_METADATA" "-DSQLITE_ENABLE_DBSTAT_VTAB" "-DSQLITE_ENABLE_FTS3" "-DSQLITE_ENABLE_FTS3_PARENTHESIS" "-DSQLITE_ENABLE_FTS5" "-DSQLITE_ENABLE_JSON1" "-DSQLITE_ENABLE_LOAD_EXTENSION=1" "-DSQLITE_ENABLE_MEMORY_MANAGEMENT" "-DSQLITE_ENABLE_RTREE" "-DSQLITE_ENABLE_STAT2" "-DSQLITE_ENABLE_STAT4" "-DSQLITE_SOUNDEX" "-DSQLITE_THREADSAFE=1" "-DSQLITE_USE_URI" "-DHAVE_USLEEP=1" "-D_POSIX_THREAD_SAFE_FUNCTIONS" "-DSQLITE_EXTRA_INIT=core_init" "-o" "/target/release/build/libsql-ffi-f1b295f639e0228d/out/bc781dd5da32bda9-extension.o" "-c" "bundled/sqlean/uuid/extension.c" with args cc did not execute successfully (status code exit status: 1).cargo:warning=In file included from bundled/src/sqlite3-sqlean-generated.c:7:
#0 6.992   cargo:warning=bundled/sqlean/crypto/extension.h:9:10: fatal error: sqlite3ext.h: No such file or directory
#0 6.992   cargo:warning=    9 | #include "sqlite3ext.h"
#0 6.992   cargo:warning=      |          ^~~~~~~~~~~~~~
#0 6.992   cargo:warning=compilation terminated.
#0 6.992 
#0 6.992   exit status: 0
#0 6.992   exit status: 0
#0 6.992   exit status: 1
#0 6.992   cargo:warning=ToolExecError: Command "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "bundled/sqlean/" "-std=c11" "-DSQLITE_CORE" "-DSQLITE_DEFAULT_FOREIGN_KEYS=1" "-DSQLITE_ENABLE_API_ARMOR" "-DSQLITE_ENABLE_COLUMN_METADATA" "-DSQLITE_ENABLE_DBSTAT_VTAB" "-DSQLITE_ENABLE_FTS3" "-DSQLITE_ENABLE_FTS3_PARENTHESIS" "-DSQLITE_ENABLE_FTS5" "-DSQLITE_ENABLE_JSON1" "-DSQLITE_ENABLE_LOAD_EXTENSION=1" "-DSQLITE_ENABLE_MEMORY_MANAGEMENT" "-DSQLITE_ENABLE_RTREE" "-DSQLITE_ENABLE_STAT2" "-DSQLITE_ENABLE_STAT4" "-DSQLITE_SOUNDEX" "-DSQLITE_THREADSAFE=1" "-DSQLITE_USE_URI" "-DHAVE_USLEEP=1" "-D_POSIX_THREAD_SAFE_FUNCTIONS" "-DSQLITE_EXTRA_INIT=core_init" "-o" "/target/release/build/libsql-ffi-f1b295f639e0228d/out/4729846b5ebdc3fd-sqlite3-sqlean-generated.o" "-c" "bundled/src/sqlite3-sqlean-generated.c" with args cc did not execute successfully (status code exit status: 1).
#0 6.992 
#0 6.992   --- stderr
#0 6.992 
#0 6.992 
#0 6.992   error occurred: Command "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "bundled/sqlean/" "-std=c11" "-DSQLITE_CORE" "-DSQLITE_DEFAULT_FOREIGN_KEYS=1" "-DSQLITE_ENABLE_API_ARMOR" "-DSQLITE_ENABLE_COLUMN_METADATA" "-DSQLITE_ENABLE_DBSTAT_VTAB" "-DSQLITE_ENABLE_FTS3" "-DSQLITE_ENABLE_FTS3_PARENTHESIS" "-DSQLITE_ENABLE_FTS5" "-DSQLITE_ENABLE_JSON1" "-DSQLITE_ENABLE_LOAD_EXTENSION=1" "-DSQLITE_ENABLE_MEMORY_MANAGEMENT" "-DSQLITE_ENABLE_RTREE" "-DSQLITE_ENABLE_STAT2" "-DSQLITE_ENABLE_STAT4" "-DSQLITE_SOUNDEX" "-DSQLITE_THREADSAFE=1" "-DSQLITE_USE_URI" "-DHAVE_USLEEP=1" "-D_POSIX_THREAD_SAFE_FUNCTIONS" "-DSQLITE_EXTRA_INIT=core_init" "-o" "/target/release/build/libsql-ffi-f1b295f639e0228d/out/4729846b5ebdc3fd-sqlite3-sqlean-generated.o" "-c" "bundled/src/sqlite3-sqlean-generated.c" with args cc did not execute successfully (status code exit status: 1).
#0 6.992 
#0 6.992 
#0 6.992 warning: build failed, waiting for other jobs to finish...
------
Dockerfile:34
--------------------
  33 |     ARG ENABLE_FEATURES=""
  34 | >>> RUN if [ "$ENABLE_FEATURES" == "" ]; then \
  35 | >>>         cargo build -p libsql-server --release ; \
  36 | >>>     else \
  37 | >>>         cargo build -p libsql-server --features "$ENABLE_FEATURES" --release ; \
  38 | >>>     fi
  39 |     RUN cargo build -p bottomless-cli --release
--------------------
ERROR: failed to solve: process "/bin/sh -c if [ \"$ENABLE_FEATURES\" == \"\" ]; then         cargo build -p libsql-server --release ;     else         cargo build -p libsql-server --features \"$ENABLE_FEATURES\" --release ;     fi" did not complete successfully: exit code: 101
```

--- a/libsql-ffi/build.rs
+++ b/libsql-ffi/build.rs
@@ -274,6 +274,7 @@ pub fn build_bundled(out_dir: &str, out_path: &Path) {
         cfg.file(&sqlean);
 
         cfg.include(format!("{BUNDLED_DIR}/sqlean/"));
+        cfg.include(format!("{BUNDLED_DIR}/src/"));
     }
 
     if cfg!(feature = "wasmtime-bindings") {

